docs/js/custom.js

```diff
@@ -97,26 +97,44 @@ $(document).ready(function() {
                     }
                 }
             });
+            let link = $('.ds-dropdown-menu a.search-page-link');
+            if (!link.length) {
+                link = $('.ds-dropdown-menu').append('<a class="search-page-link" href="" style="position: absolute; bottom: 0px; z-index: 1000;">See all results</a>');
+            }
+            let href = '/en/' + branchName + '/search_results/#q=' + encodeURI($(/*this.inputSelector*/'#search_input').val()) + '&p=1';
+            link.attr('href', href).show();
+            if (hits.length < 10/*hitsPerPage*/) {
```

Comment:
hits.length == hitsPerPage means that there are hitsPerPage or more results; hits.length > hitsPerPage can't happen. So we can only be sure that the link must be hidden if there is less than hitsPerPage results.